shared_types/resource.ts

```diff
@@ -0,0 +1,12 @@
+import { z } from "zod";
+import { scamCategory } from "./scamCategory";
+
+export const resourceSchema = z.object({
+  _id: z.string(),
+  category: scamCategory,
+  content: z.string().max(500),
+  links: z.array(z.string()),
+  image: z.instanceof(Buffer).optional(),
```

Comment:
This gives me an error:

1. Cannot find name 'Buffer'. Do you need to install type definitions for node? Try `npm i --save-dev @types/node`. [2580]

This was after `npm i`. Must we add `@types/node` as a dev dependency?

Reply:
Ah yes I might have done that